nixos/modules/system/boot/luksroot.nix

```diff
@@ -414,12 +509,6 @@ in
                     description = "The filesystem of the unencrypted device.";
                   };
 
-                  mountPoint = mkOption {
```

Comment:
Please add a release-note about the removal of `boot.initrd.luks.devices.<name?>.yubikey.storage.mountPoint`, and the expected upgrade path for a user that previously used this option.